packages/pyxis-react/src/components/Accordion/Item.tsx

```diff
@@ -0,0 +1,118 @@
+import React, {
+  FC, ReactNode, useEffect, useRef, useState,
+} from 'react';
+import classNames from 'classnames';
+import { IconChevronDown } from 'components/Icons';
+import { Variant } from './Accordion';
+
+const setClass = (alt?:boolean, color?:Variant, classname?:string):string => classNames(
```

Comment:
questa variabile può diventare direttamente variant.
Tutte le funzioni di classe si chiamano `getClasses`
```suggestion
const getClasses = (alt?:boolean, variant?:Variant, classname?:string):string => classNames(
```